SingularityUI/app/handlebarsHelpers.coffee

```diff
@@ -106,3 +106,6 @@ Handlebars.registerHelper 'usernameFromEmail', (email) ->
 
 Handlebars.registerHelper 'substituteTaskId', (value, taskId) ->
     value.replace('$TASK_ID', taskId)
+
+Handlebars.registerHelper 'trimS3File', (filename) ->
+    return '...' + filename.substring(filename.indexOf("service.log"));
```

Comment:
I was afraid that was the case. This model doesn't include the task id but I should be able to add it in.